provider/resource_collection.go

```diff
@@ -33,7 +34,7 @@ func validateFieldTypes() schema.SchemaValidateFunc {
 func ResourceCollection() *schema.Resource {
```

Comment:
example of using with acl, in case we will need to change permissions:
```

resource "itsi_splunk_collection" "engr_aws_accounts" {
  provider = itsi.engr
  name     = local.aws_accounts_collection
  app      = local.resources_shared_app
}

resource "splunk_generic_acl" "engr_generic_acl" {
  provider = splunk.engr
  path     = "storage/collections/config/${itsi_splunk_collection.engr_aws_accounts.name}"
  acl {
    owner   = itsi_splunk_collection.engr_aws_accounts.owner
    app     = itsi_splunk_collection.engr_aws_accounts.app
    sharing = "global"
    read    = ["*"]
    write   = ["admin", "power", "rest-role"]
  }
}

```

Reply:
Yes, this is partially why I did not go forward with my branch, I found out there was a generic acl resource that's actually easier to use than adding some acl logic within the collection resource. 

Too bad I remember that only now...

Reply:
I knew about this resource from a comment in your PR in the first place :) 
> Too bad I remember that only now...
but left everything to follow easy